Source/Schema.NET/Thing.Partial.cs

```diff
@@ -34,8 +35,35 @@ public partial class Thing : JsonLdObject
             },
             DefaultValueHandling = DefaultValueHandling.Ignore,
             NullValueHandling = NullValueHandling.Ignore,
+            DateParseHandling = DateParseHandling.DateTimeOffset,
             StringEscapeHandling = StringEscapeHandling.EscapeHtml
         };
+        
+        
+        /// <summary>
+        /// Serializer settings used when deserializing.
+        /// </summary>
+        private static readonly JsonSerializerSettings DeserializerSettings = new JsonSerializerSettings()
+        {
+            Converters = new List<JsonConverter>()
+            {
+                new StringEnumConverter()
+            },
+            NullValueHandling = NullValueHandling.Ignore,
+            DefaultValueHandling = DefaultValueHandling.Ignore,
+            DateParseHandling = DateParseHandling.DateTimeOffset,
+            // The ASP.NET MVC framework defaults to 32 (so 32 levels deep in the JSON structure)
+            // to prevent stack overflow caused by malicious complex JSON requests.
+            MaxDepth = 32

```

Comment:
Apart from MaxDepth this is the same as the one above. Is there a reason to keep them separate?

Reply:
No reason to keep them separate, MaxDepth could be applied to both (though not really necessary for serialisation)